src/www/+page.svelte

```diff
@@ -55,7 +55,7 @@
 				>Components</Button
 			>
 			&nbsp;
-			<Button href="{base}/docs/layouts/card" size="xl">Layouts</Button>
+			<Button href="{base}/docs/layouts/card" size="xl" id="layout_button">Layouts</Button>
```

Comment:
@GlistenSTAR are you sure Svelte allows you to setup the `id`? I thought, it's recommended to use classes, but please correct me if I'm wrong.

Reply:
yes, id is possible to change the css.
We can use it.